src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -0,0 +1,190 @@
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final InventoryItemResource nod = itemsFixture.basedUponNod();
+
+    final IndividualResource steve = usersFixture.steve();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    loansFixture.checkOutByBarcode(nod);
+
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
+    createRequest(nod, rebecca, secondServicePointId);
+
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+    loansFixture.checkInByBarcode(nod);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    assertThat(items.size(), is(2));
+  }
+
+  @Test
+  public void reportExcludesItemsOtherThenInTransitStatus()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final InventoryItemResource nod = itemsFixture.basedUponNod();
+
+    final IndividualResource steve = usersFixture.steve();
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    loansFixture.checkOutByBarcode(nod);
+
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
+
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    verifyResponse(smallAngryPlanet, secondServicePointId, items);
+  }
+
+  @Test
+  public void reportIncludeItemsInTransitIrrespectiveOfServicePoint()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final InventoryItemResource nod = itemsFixture.basedUponNod();
+
+    final IndividualResource steve = usersFixture.steve();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    final UUID firstServicePointId = servicePointsFixture.cd1().getId();
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    loansFixture.checkOutByBarcode(nod);
+
+    createRequest(smallAngryPlanet, steve, firstServicePointId);
+    createRequest(nod, rebecca, secondServicePointId);
+
+    loansFixture.checkInByBarcode(new CheckInByBarcodeRequestBuilder()
+      .forItem(smallAngryPlanet)
+      .on(DateTime.now(DateTimeZone.UTC))
+      .at(secondServicePointId));
+    loansFixture.checkInByBarcode(nod);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    assertThat(items.size(), is(2));
```

Comment:
Does the report include information about the destination service point?

If it does, I think this test should check that the right information is included for each item

Reply:
This information will be cover in related story https://issues.folio.org/browse/CIRC-519

Reply:
The scenarios in this story includes `Check in service point`, `Check in date and time` and `Destination service point` are the first two deferred to another story?

The latter, `inTransitDestinationServicePointId` is included in the code for this story, hence I think it should be tested as part of this change

Reply:
Alas, it appears the change has been merged before I had chance to provide that feedback

Reply:
Yeah, it really doesn't make sense to add code in one PR and add tests for it in another PR which may not even be merged in this sprint.

Reply:
Agree, however it blocks the story https://issues.folio.org/browse/CIRC-519 which is almost ready. And I think it can be possible to do this in scope of the CIRC-519 story because both of them aim to create report.